SimplenoteScreenshots/SimplenoteScreenshots.swift

```diff
@@ -209,46 +208,19 @@ class SimplenoteScreenshots: XCTestCase {
         XCTAssertTrue(passcodeCell.waitForExistence(timeout: 3))
         passcodeCell.tap()
 
-        let expectation = XCTNSPredicateExpectation(
-            predicate: NSPredicate { _, _ in self.isPasscodeScreenVisible(using: app) },
-            object: .none
-        )
-        wait(for: [expectation], timeout: 5)
+        return try PasscodeScreen(app: app)
     }
 
     func dismissPasscodeScreenIfNeeded(using app: XCUIApplication) {
-        guard isPasscodeScreenVisible(using: app) else { return }
-        typeOnPassCodeScreen(1234, using: app)
-    }
-
-    func isPasscodeScreenVisible(using app: XCUIApplication) -> Bool {
-        // Check there's a button for each line of the custom number keypad.
-        //
-        // We only need to wait for the first number, if that's on screen, then the others should be
-        // on screen already, too.
-        return app.staticTexts["1"].waitForExistence(timeout: 3)
-            && app.staticTexts["4"].exists
-            && app.staticTexts["7"].exists
-            && app.staticTexts["0"].exists
-    }
+        guard PasscodeScreen.isLoaded(in: app) else { return }
 
-    func typeOnPassCodeScreen(_ input: Int, using app: XCUIApplication) {
-        // This converts an Int into an [Int] of its digits
-        let digits = "\(input.magnitude)".compactMap(\.wholeNumberValue)
-
-        digits.forEach { digit in
-            let input = app.staticTexts["\(digit)"]
-            XCTAssertTrue(input.waitForExistence(timeout: 3))
-            // Both the custom UIButton and its UILabel match the "\(digit)" query. We need to pick
-            // one for the tap to work.
-            input.firstMatch.tap()
+        do {
+            try PasscodeScreen(app: app).type(passcode: 1234)
+        } catch {
+            XCTFail("Expected passcode screen to exist but it did not.")
         }
     }
 
-    func type(_ text: String, onFirstKeyboardOf app: XCUIApplication) {
-        text.forEach { app.keyboards.firstMatch.keys[String($0)].tap() }
-    }
-
```

Comment:
This was unused. Likely a leftover from the time the passcode screen used the system's numerical keyboard.